### PR TITLE
Add Bundle and Relationship functional tests

### PR DESCRIPTION
**Pull request by oshtaier**

### CODE REVIEW

File: tests/functional/tests/test_ctia_api.py [hunk 1]
```diff
@@ -271,6 +272,75 @@ def test_python_module_ctia_positive_bulk(module_headers, module_tool_client):
...
+    assert post_tool_response[
+        'indicators'][0]['id'] == indicator_post_response['id']
+    # Validate import endpoint
+    payload = {
+        'schema_version': SERVER_VERSION,
+        'type': 'bundle',
+        'source': 'random source',
+    }
+    post_tool_response = bundle.import_.post(payload=payload)
+    assert post_tool_response
```
Comment (cawabanga2811) [inline tests/functional/tests/test_ctia_api.py]: what did this assert do?

Comment (oshtaier) [reply tests/functional/tests/test_ctia_api.py]: verify that response is not empty

Comment (cawabanga2811) [reply tests/functional/tests/test_ctia_api.py]: is this enough in this case, how do you think?

Comment (oshtaier) [reply tests/functional/tests/test_ctia_api.py]: I am not happy about that, but we have requirement to work with minimal datasets, so I can't request for some more detailed input data.
We will get something like {'result: []'}, so I decided to validate that we will not get at least `b''`. Anyway custom python module will fail for empty response data, so basically my validation is just sanity check

**Pull request merged**